### ISSUE TITLE
How to navigate with keyboard on tool customization view?

### ISSUE THREAD

**Issue by aiday-mar** [microsoft/vscode#323427]
Testing https://github.com/microsoft/vscode/issues/323371

- Open the tool customization view
- Select a tool using the tab 
- How to go to the tools from the category selected? Left, right, up and down arrows don't seem to work.

**Comment by dmitrivMS** [comment]
Current it's just tab, but I agree it should behave more like tree view.
